download-latest.sh

```diff
@@ -168,16 +168,17 @@ failure_usage() {
 
 # MAIN
 latest="$(get_latest)"
-get_os
-if [ "$?" -eq 1 ]; then
+
+if ! get_os; then
```

Comment:
I suppose this is fair, we were only testing that the `get_os` out was `1` instead of checking that it is simply different from `0`, so...